### PR TITLE
Add copyright & license

### LINKED ISSUE #2196: Add copyright and license information to About window
We should mention our copyright and our (new) license in the APInf About window (about.html).

# To do
* [ ] Add copyright text to About window 
* [ ] Add license information to About window

# Proposal:
- **Copyright** 2017 Apinf Oy
- **License** Licensed under [EUPL]( https://joinup.ec.europa.eu/software/page/eupl) v.1.1

### PR DESCRIPTION
**Pull request by marla-singer**
Closes #2196

### CODE REVIEW

File: about/client/about.html [hunk 1]
```diff
@@ -38,6 +38,23 @@ <h2 class="modal-title" id="apiBacklogFormModalLabel">
...
+              {{_ "aboutApinf_copyright" }}
+            </dt>
+            <dd>
+              2017 Apinf Oy
+            </dd>
+            <dt>
+              {{_ "aboutApinf_license" }}
+            </dt>
+            <dd>
+              Licensed under <a href="https://joinup.ec.europa.eu/software/page/eupl" target="_blank">EUPL</a> v.1.1
```
Comment (brylie) [inline about/client/about.html]: The text 'Licensed under' can be removed, since the table key already says 'License'.

Comment (brylie) [inline about/client/about.html]: The text 'Licensed under' can be removed, since the table key already says 'License'.

Comment (marla-singer) [inline about/client/about.html]: Done

**Comment by marla-singer** [comment]
@nnn Ready

**Pull request merged**